app/controllers/owners_controller.rb

```diff
@@ -0,0 +1,38 @@
+class OwnersController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @owners = Owner.order(created_at: :desc)
+  end
+
+  def new
+    @user  = User.new
+    @owner = @user.build_owner
+  end
+
+  def create
+      @user  = User.new
```

Comment:
In this `create` method, you never save the new `user`. Do you need to save the new `user` in order to save the `owner`?

Reply:
It was something that I never understood before. But I've fixed it now with John's help today. Looks like "current_user" should have only one "owner" that can be created by using `build_owner` method. In an overall, I ended up not creating a new user, instead, the current_user is creating the owner.

Reply:
Looks good!